cyberham/bot/leaderboard.py

```diff
@@ -55,7 +55,7 @@ async def leaderboard_search(interaction: discord.Interaction, activity: str):
             return
 
         prev, curr = 0, 0
-        leaderboard = sorted(leaderboard, key=lambda kv: (kv[1], kv[0]), reverse=True)
+        leaderboard = sorted(leaderboard, key=lambda kv: (kv[1], kv[0]), reverse=False)
```

Comment:
```suggestion
        leaderboard = sorted(leaderboard, key=lambda kv: (-kv[1], kv[0]), reverse=False)
```

If you add the - for the points value, it will properly sort it with points descending but names alphabetical. Right now it has names alphabetical but points in reverse, which is not quite the desired behavior.